### PR TITLE
Refactored stripe keys to use dicts

### LINKED ISSUE #1288: Add stripe key for trikom
Trikom needs its own stripe key as demonstrated today with the Star Wars event. I had to use arrkoms key today as I got this on short notice and prokoms key was still in test mode (otherwise we could have switched and added them again later).

### LINKED ISSUE #1310: Change stripe key data structure to dict
Having more than one set of stripe keys means clutter if they are stored in a list. A much safer way is to use a dict for that kind of storage. Not only is it safer, but also a lot more understandable in code.

```python
key = STRIPE_KEYS[0] vs STRIPE_KEYS['arrkom']
```

### PR DESCRIPTION
**Pull request by Torrib**
Fixes #1288, fixes #1310

### CODE REVIEW

File: apps/payment/models.py [hunk 1]
```diff
@@ -27,16 +27,16 @@ class Payment(models.Model):
         (3, _(u'Utsettelse')),
     )
 
-    STRIPE_KEY_CHOICES = (
-        (0, "Arrkom"),
-        (1, "Prokom"),
-        (2, "Trikom"),
-    )
+    #creates tuples used as key choices.
+    STRIPE_KEY_CHOICES = zip(settings.STRIPE_PUBLIC_KEYS.keys(),settings.STRIPE_PUBLIC_KEYS.keys())
```
Comment (hernil) [inline apps/payment/models.py]: Where does the keys() method come from?

Comment (Torrib) [inline apps/payment/models.py]: Buildt in dictionary functionality

File: apps/payment/migrations/0013_auto_20160210_2010.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+    operations = [
+        migrations.RemoveField(
+            model_name='payment',
+            name='stripe_key_index',
+        ),
+        migrations.AddField(
+            model_name='payment',
+            name='stripe_key',
+            field=models.CharField(default=b'arrkom', max_length=10, verbose_name='stripe key', choices=[(b'arrkom', b'pk_test_replace_this'), (b'prokom', b'pk_test_Ur8B7E5uvheMlpOUVu9SGGbn'), (b'trikom', b'pk_test_bziEErlw9KmTHcNeC6cBXHhw')]),
```
Comment (sklirg) [inline apps/payment/migrations/0013_auto_20160210_2010.py]: Keys to the public?

File: onlineweb4/settings/example-local.py [hunk 3]
```diff
@@ -146,17 +146,17 @@
...
+STRIPE_PUBLIC_KEYS = {
+    "arrkom": "pk_test_replace_this",
+    "prokom": "pk_test_replace_this",
+    "trikom": "pk_test_replace_this"
+}
+
+STRIPE_PRIVATE_KEYS = {
+    "arrkom": "sk_test_replace_this",
+    "prokom": "pk_test_replace_this",
+    "trikom": "pk_test_replace_this"
```
Comment (sklirg) [inline onlineweb4/settings/example-local.py]: One is prefixed sk and the others are prefixed pk? Be consistent please, so people who want to debug this uses the correct key.

**Comment by sklirg** [comment]
Looks good to me. :+1:

**Pull request merged**